TicTacToeApp/ViewController.swift

```diff
@@ -8,18 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
+  var currentPlayerMark = "X"
```

Comment:
This comment is old. I think it makes sense to initialize where you have it. Alternatively, you could have the `Game` be responsible for determining the initial player mark.